editor/editor_node.cpp

```diff
@@ -3090,6 +3081,21 @@ void EditorNode::_menu_option_confirm(int p_option, bool p_confirmed) {
 	}
 }
 
+String EditorNode::node_scene_name_casing(String &root_name) {
```

Comment:
```suggestion
String EditorNode::adjust_scene_name_casing(String &root_name) {
```
To be consistent with [another similar function](https://github.com/godotengine/godot/blob/4ba934bf3d1e697d8f332b5e8cfd694cdf49a7ba/scene/main/node.cpp#L945).

Reply:
Can also be `const String &`.

Reply:
Would `const ` be required when I'm altering `p_root_name` by passing it to either `to_pascal_case `or `to_snake_case`?

Reply:
both of the functions I'm passing to expect and un-const string

Reply:
You don't really need to alter the param, just do `return p_root_name.to_pascal_case()` etc.

Reply:
easy as, cheers mate